### DISCUSSION TITLE
[灌水]大新闻!珀尔曼跑了!

### DISCUSSION THREAD

**Discussion by luckey-lang** [share121/inter-knot#246] [其他]
![IMG_20240809_225418](https://github.com/user-attachments/assets/a8e98880-94d3-477f-a359-edfca5b8430a)天啊大家看新闻了吗，珀尔曼在远景案的开庭路上逃跑了!新闻说飞艇的信号飞到外环之后消失了，治安局已经发通缉令了!

**Comment by 43-13** [comment]
看到了大震惊

**Comment by NE-Janus** [comment]
请市民们不要担心，我市治安局已经出动了，远景案暂时搁置，我们绝不姑息罪犯！

**Comment by Luckyxinchen** [comment]
绝不能放过他啊（生气）

**Comment by Tianyyzhan** [comment]
感觉可以拍一场电影了

**Comment by lllliufeng** [comment]
赏金挺高的吧
